### DISCUSSION TITLE
Close and reopen WinEdt 11.2, all formatting restored to defaults

### DISCUSSION THREAD

**Discussion by Qwer-star-cell** [WinEdt-Team/WinEdt-Support#47] [1.2 Bugs and Issues]
Recently, I just downloaded and installed WinEdt 11.2, but I've encountered some problems. Every time I set the fonts, shortcuts, and theme format, when I close WinEdt and open it again, I find that it reverts back to the default format, what's wrong with it?

**Comment by Alex-Simonic** [comment]
Problems like this are usually caused by "bizarre" custom preferences during the installation (like installation for all users in the Program Files folder without personal profile creation).

Start Options -> Configuration Wizard. On the first page, you will find Install Folder and Local Folder. 

  1. What is the Local Folder and does it exist?
  2. Are you allowed to write there?
  3. Is a file WinEdt.dnt there and does its date change when you close WinEdt?

Let us know.

**Reply by Hu-26** [reply]
Hello, Alex-Simonic! 
I'm also having the same problem, here's my Configuration Wizard interface, I hope you can help me solve this problem, thank you very much!
![1](https://github.com/user-attachments/assets/5c97e4ac-bb3d-4d93-86ac-23c83657f12a)

**Reply by Alex-Simonic** [reply]
To avoid duplication use [Discussion #76](https://github.com/WinEdt-Team/WinEdt-Support/discussions/76). This is the same issue related to the local folder (%b)...
